src/test/java/org/eclipse/yasson/internal/AnnotationIntrospectorTest.java

```diff
@@ -59,6 +63,23 @@ public void testObjectShouldBeCreateableFromJsonbAnnotatedStaticFactoryMethodIgn
         assertCreatedInstanceContainsAllParameters(ObjectWithJsonbCreatorAndConstructorPropertiesAnnotation.example(), creator);
     }
 
+    @Test
+    public void testJsonbAnnotatedProtectedConstructorLeadsToAnException() {
+        exception.expect(JsonbException.class);
+        exception.expectCause(IsInstanceOf.instanceOf(IllegalAccessException.class));
+        JsonbCreator creator = instrospector.getCreator(ObjectWithJsonbCreatorAnnotatedProtectedConstructor.class);
+        assertCreatedInstanceContainsAllParameters(ObjectWithJsonbCreatorAnnotatedProtectedConstructor.example(), creator);
+    }
+
+    // TODO Under discussion. Shouldn't this test work?
```

Comment:
so currently this test fails with an IllegalAccessException? If so, please open up another issue and link to the URL in the comment here so we know why the test is ignored

Reply:
done.
i opened https://github.com/eclipse-ee4j/yasson/issues/326.